docs/services/slam/cartographer/_index.md

```diff
@@ -357,125 +453,66 @@ See [Attributes](#attributes) for details.
 | `existing_map` | string | Optional | The alias of the package containing the existing map to build on (in "Update existing map" mode) or localize on (in "Localize only" mode). |
 | `config_params` |  obj | Optional | Parameters available to fine-tune the `cartographer` algorithm: [read more below](#config_params). |
 
-### View the Map
-
-#### "Create new map" mode
-
-Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
-
-Enter a name for your new map and click `"Start session"`, then wait for the slam session to finish starting up in the cloud, which **takes about 2 minutes**.
-
-Make sure to either **manually refresh**, or **change the refresh frequency** to something other than `Manual`.
+2. Start a mapping session
 
-![slam RC card start session](/services/slam/slam-RC-card-start-session.png)
+   Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
 
-While the slam session is starting, you will see a loading screen.
+   Unlike when creating or updating a map, you do not need to start and end a slam session.
+   The pointcloud for the existing map will appear **immediately** and Cartographer will try to find your robot's position on it.
 
-![slam RC card wait for session to finish starting](/services/slam/slam-RC-card-wait-for-session-to-finish-starting.png)
+   Since the map will not change, nothing new will be added to this robot's location's **SLAM library**.
 
-Once the slam session has finished starting, your first pointcloud will appear.
+   ![slam RC card localize only](/services/slam/slam-RC-card-localize-only.png)
 
-![slam RC card first pointcloud](/services/slam/slam-RC-card-first-pointcloud.png)
+   {{% alert title="Info" color="info" %}}
 
-You will be able to see that your cloud slam session is in progress from your **Location** page's **SLAM library** tab.
+   Cartographer may take several minutes to find your robot's position on the existing map.
+   In the meantime, your robot will show up at the map's origin (with the `(x,y)` coordinates `(0,0)`).
 
-![offline mapping maps computing table](/services/slam/offline-mapping-maps-computing-table.png)
+   If you move your robot, it will appear to be moving in a trajectory from the map's origin.
 
-When you would like to end the slam session, back on the robot's **Control** tab, click `"End session"`. If you do not click `"End session"`, the slam session will automatically be ended after 45 minutes.
+   {{% /alert %}}
 
-Once the session has ended, the map will be saved to your **Location** page's **SLAM library** tab.
+## Create a new map from cloud data
```

Comment:
[optional] "Create or update a map from previously captured data"? (mention both create and update, and say "previously captured" rather than "cloud")

The reason I'm suggesting "previously captured" instead of "cloud" is that even when using a live robot, the lidar/imu data gets sent to the cloud

Reply:
The UI should say that too then, right?

Reply:
Ahh, I see now. If you click "Make new map" the UI says "Create a new map from cloud data", and if you click "Update map" it says "Update <map name> from cloud data" (for example "Update whole-office-3 from cloud data").

How about "Create or update a map from cloud data"?

Or, feel free to leave it as you see fit and resolve

Reply:
Changing it to your first suggestion - we should probably just update the UI when we can as well

Reply:
Sounds good, filed a ticket to change the UI: https://viam.atlassian.net/browse/RSDK-5423

Feel free to resolve!